samples/Integration/CMake-fetch-content-app/README.md

```diff
@@ -0,0 +1,42 @@
+# Integrating the Azure SDK for C++ with CMake fetch content to your application
+
+This application is an example of integrating the Azure SDK for C++ to your application. It uses the CMake fetch content functionality to automatically clone the Azure SDK for C++ repo to your `build` directory. This approach is useful if you are using CMake to build your application.
+
+## Pre-requisites
+
+Make sure to install the [Azure SDK for C++ dependencies](https://github.com/Azure/azure-sdk-for-cpp/blob/master/CONTRIBUTING.md#third-party-dependencies) first.
+
+- CMake project.
```

Comment:
Specify the version of CMake that is required, add links to CMake for those that don't have it installed